Sources/NetworkService/NetworkError.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+/// Contains network errors that may occur, while the application is running.
+public enum NetworkError: Error, CaseIterable {
+    /// Indicates that the Internet connection has been lost.
+    case lostConnection
+    
```

Comment:
```suggestion

```
Can you run SwiftFormat for the package to adhere to the conventions of formatting?

Reply:
Sorry for this. Fixed it.

Reply:
If I will delete this empty line, code will became hard to read.

Reply:
The reason isn't empty line but indentations on the empty lines.

Reply:
Indents in empty lines have been removed.